video/reverse_video_search/1_reverse_video_search_engine.ipynb

```diff
@@ -312,7 +314,7 @@
     "\n",
     "- `.video_decode.ffmpeg['path', 'frames']()`: an embeded Towhee operator reading video as frames with specified sample method and number of samples. [learn more](https://towhee.io/video-decode/ffmpeg)\n",
     "\n",
-    "- `.video_classification['frames', 'vec'].pytorchvideo()`: an embeded Towhee operator applying specified model to video frames, which can be used to generate video embedding. [learn more](https://towhee.io/video-classification/pytorchvideo)\n",
+    "- `.action_classification['frames', 'vec'].pytorchvideo()`: an embeded Towhee operator applying specified model to video frames, which can be used to generate video embedding. [learn more](https://towhee.io/action-classification/pytorchvideo)\n",
```

Comment:
I think the parameter should modify to `'frames', ('labels', 'scores', 'features')`